.github/workflows/tox.yml

```diff
@@ -0,0 +1,212 @@
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+      FORCE_COLOR: 1
+
+    steps:
+    - name: Check out src from Git
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0  # needed by setuptools-scm
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    # - name: set PY_SHA256
+    #   run: echo "::set-env name=PY_SHA256::$(python -VV | sha256sum | cut -d' ' -f1)"
+    # - name: Pre-commit cache
+    #   uses: actions/cache@v1
+    #   with:
+    #     path: ~/.cache/pre-commit
+    #     key: ${{ runner.os }}-pre-commit-${{ env.PY_SHA256 }}-${{ hashFiles('setup.cfg') }}-${{ hashFiles('tox.ini') }}-${{ hashFiles('pyproject.toml') }}-${{ hashFiles('.pre-commit-config.yaml') }}-${{ hashFiles('pytest.ini') }}
+    # - name: Pip cache
+    #   uses: actions/cache@v1
+    #   with:
+    #     path: ~/.cache/pip
+    #     key: ${{ runner.os }}-pip-${{ env.PY_SHA256 }}-${{ hashFiles('setup.cfg') }}-${{ hashFiles('tox.ini') }}-${{ hashFiles('pyproject.toml') }}-${{ hashFiles('.pre-commit-config.yaml') }}-${{ hashFiles('pytest.ini') }}
+    #     restore-keys: |
+    #       ${{ runner.os }}-pip-
+    #       ${{ runner.os }}-
+    - name: Install tox
+      run: |
+        python3 -m pip install --upgrade tox
+    - name: Log installed dists
+      run: >-
+        python -m pip freeze --all
+    - name: >-
+        Initialize tox envs
+      run: >-
+        python -m
+        tox
+        --parallel auto
+        --parallel-live
+        --notest
+        --skip-missing-interpreters false
+        -vv
+      env: ${{ matrix.env }}
+    - name: Test with tox
+      run: |
+        python -m tox --parallel auto --parallel-live
+      env: ${{ matrix.env }}
+    - name: Archive logs
+      uses: actions/upload-artifact@v2
+      with:
+        name: logs.zip
+        path: .tox/**/log/
+
+  unit:
+    name: >-
+      ${{ matrix.tox_env }}@${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      # fail-fast: false
+      # max-parallel: 5
+      # The matrix testing goal is to cover the *most likely* environments
+      # which are expected to be used by users in production. Avoid adding a
+      # combination unless there are good reasons to test it, like having
+      # proof that we failed to catch a bug by not running it. Using
+      # distribution should be prefferred instead of custom builds.
+      matrix:
+        python-version:
+        # keep list sorted as it determines UI order too
+        - 3.6
+        - 3.7
+        - 3.8
+        - 3.9
+        # NOTE: Installing ansible under 3.10-dev is currently not
+        # NOTE: possible because compiling cffi explodes.
+        os:
+        # https://help.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners
+        - ubuntu-20.04
+        # - windows-latest
+        # - windows-2016
+        include:
+        - tox_env: py36
```

Comment:
What's the point of having the toxenv for each interpreter if `tox -e py` picks up the current one already?